okapi-core/src/test/java/org/folio/okapi/ModuleTest.java

```diff
@@ -141,7 +141,7 @@ public ModuleTest(String value) throws Exception {
         break;
       case "mongo":
         if (mongoDBContainer == null) {
-          mongoDBContainer = new MongoDBContainer("mongo:3.6.20");
+          mongoDBContainer = new MongoDBContainer("mongo:3.6.23");
```

Comment:
https://www.mongodb.com/support-policy/lifecycles

MongoDB 3.6 End of Life Date April 2021

The container will disappear from https://hub.docker.com/_/mongo soon.